assets/css/index.css

```diff
@@ -162,6 +162,11 @@ hr {
   vertical-align: sub;
 }
 
+.navbar .sidebar-toggler, .close-sidebar{
+    outline:none;
```

Comment:
It seems like on `.close-sidebar:active` this is overridden by another rule, so please add `!important` to `outline: none`

Reply:
If by overridden you mean that on clicking the .close-sidebar button, the section slides to the left and instantly gets back in the original position. Is it supposed to happen? @shahednasser

Reply:
No, I mean that on click the `.close-sidebar` button becomes focused and due to CSS rules made by bootstrap, an outline is added. To make sure our CSS rules are applied instead of bootstrap's, add `!important` to the rule

Reply:
Ill add the changes